docs/services/data/configure-data-query.md

```diff
@@ -0,0 +1,124 @@
+   Provide your organization's `org-id` from step 2, and a desired new password for your database user.
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam data database configure --org-id=<YOUR-ORGANIZATION-ID> --password=<NEW-DBUSER-PASSWORD>
+   ```
+
+   This command configures a new database user for your org for use with data query.
+   If you have already created this user, this command updates the password for that user instead.
+
+1. Determine the hostname for your organization's MongoDB Atlas Data Federation instance.
+   Provide your organization's `org-id` from step 2:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam data database hostname --org-id=<YOUR-ORGANIZATION-ID>
+   ```
+
+   This command returns the `hostname` (including database name) you can use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+
+For more information, see the documentation for the [Viam CLI `database` command](/manage/cli/#data).
+
+## Query data
+
+Once you have [configured data query](#configure-data-query), you can directly query synced tabular data using MQL or SQL.
+
+You can use any client that is capable of connecting to MongoDB Atlas instances, including [the `mongosh` shell](https://www.mongodb.com/docs/mongodb-shell/), [MongoDB Compass](https://www.mongodb.com/docs/compass/current/), and many third-party tools.
+Use the `hostname` and `user` returned from the setup above to connect from your desired client to the MongoDB Atlas instance.
+
+For example, to connect to your Viam organization's MongoDB Atlas instance and query data using the `mongosh` shell:
+
+1. If you haven't already, [download the `mongosh` shell](https://www.mongodb.com/try/download/shell).
+   See the [`mongosh` documentation](https://www.mongodb.com/docs/mongodb-shell/) for more information.
+
+1. Run the following command to connect to the Viam organization's MongoDB Atlas instance from `mongosh`:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   mongosh "mongodb+srv://<YOUR-DB-HOSTNAME>" --apiVersion 1 --username db-user-<YOUR-ORG-ID>
+   ```
+
+   Where:
+
+   - `<YOUR-DB-HOSTNAME>` is your organization's assigned MongoDB Atlas instance hostname (including database name), as returned from `viam data database hostname` above.
+   - `<YOUR-ORG-ID>` is your organization ID as determined above.
+     The full username you provide to the `--username` flag should therefore resemble `db-user-abcdef12-abcd-abcd-abcd-abcdef123456`.
+
+1. Once connected, you can run [MQL](https://www.mongodb.com/docs/manual/tutorial/query-documents/) or SQL to query captured data directly. For example:
+
+   - The following MQL query searches the `sensorData` database and `readings` collection, and returns sensor readings from an ultrasonic sensor on a specific `robot_id` where the recorded `distance` measurement is greater than `.2` meters:
```

Comment:
Q: Did I guess correctly that `.2` is unit: meters.

Reply:
Correct!